### ISSUE TITLE
Problems with set titles and other text fields

### ISSUE THREAD

**Issue by emptyflask** [reagent/fleakr#1]
For instance:

```
>> sets[1]
=> #<Fleakr::Objects::Set:0x10210d940 @primary_photo_id="3729672314", @description="", @id="12345678901234567890", @count="33", @authentication_options={}, @title="", @document={elem <photoset farm="3" photos="33" videos="0" primary="3729672314" server="2437" id="12345678901234567890" secret="1234567890"> "\n\t\t" {elem <title> "Shake-a-spear" </title>} "\n\t\t" {emptyelem <description>} "\n\t" </photoset>}>
```

As you can see, there is a title associated with the photo, but @title is empty. Maybe Flickr changed their API? I'm seeing similar issues with photo.url and possibly other fields as well.

**Comment by reagent** [comment]
Can you give me some more context around this error?  How did you do the assignment to sets?  Is it happening with other sets?

**Comment by emptyflask** [comment]
Ok, here's a more complete example:

```
require "rubygems"
require "fleakr"

Fleakr.api_key = '12345678901234567890123456789012'
user = Fleakr.user('my_user_name')

user.sets.each do |set|
  puts set.title
end
```

I get nothing. I've tried it with another account as well (and both definitely have sets with titles).

**Comment by emptyflask** [comment]
I've looked around on some other sites, and hpricot 0.8.2 is broken.
Hpricot 0.8.1 works with Fleakr.

**Comment by reagent** [comment]
Thanks for figuring that out - I'd like to get Fleakr working with 0.8.2 since the only thing keeping me in the 0.6.x series is JRuby compatibility (which appears to be fixed in 0.8.2).  I'll open a separate issue for this.

**Comment by reagent** [comment]
I added issue #2 to track this - if you have the time and interest in working on this, let me know.

**Comment by reagent** [comment]
Another github user submitted a patch - the fix is in 0.6.2
